packages/rum-core/src/domain/rumEventsCollection/view/interactionCountPolyfill.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * performance.interactionCount polyfill
+ *
+ * The interactionCount is an integer which counts the total number of distinct user interactions,
+ * for which there was a unique interactionId.
+ *
+ * The interactionCount polyfill is an estimate based on a convention specific to Chrome. Cf: https://github.com/GoogleChrome/web-vitals/pull/213
+ * This is currently no an issue as the polyfill is only used for INP which is currently only supported on Chrome.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏typo
```suggestion
 * This is currently not an issue as the polyfill is only used for INP which is currently only supported on Chrome.
```